types/index.d.ts

```diff
@@ -626,6 +669,10 @@ declare class Client {
   [key: string]: unknown;
 
   static find(id: string): Promise<Client | undefined>;
+
+  static needsSecret(metadata: ClientMetadata): boolean;
+
+  static Schema: typeof Schema;
```

Comment:
Schema is not a public API, it may be used for hacks but it is subject to change between versions. Please remove.

Reply:
Got it. No problem. ;)

Reply:
reverted ;)